sdk/monitor/azquery/example_test.go

```diff
@@ -109,17 +120,12 @@ func ExampleLogsClient_QueryWorkspace_second() {
 	var QueryResults []queryResult
 	for _, table := range res.Tables {
 		QueryResults = make([]queryResult, len(table.Rows))
-		indexBool := table.ColumnIndexLookup["Bool"]
-		indexLong := table.ColumnIndexLookup["Long"]
-		indexDouble := table.ColumnIndexLookup["Double"]
-		indexString := table.ColumnIndexLookup["String"]
-
 		for index, row := range table.Rows {
 			QueryResults[index] = queryResult{
-				Bool:   row[indexBool].(bool),
-				Long:   int64(row[indexLong].(float64)),
-				Double: float64(row[indexDouble].(float64)),
-				String: row[indexString].(string),
+				Bool:   row[0].(bool),
```

Comment:
Are these indices conventional in some way?

Reply:
You have to access Row data by index. This example assumes you know the result format.